.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       fail-fast: true
       max-parallel: 2
       matrix:
-        node-version: [8.x, 12.x, 14.x]
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
do we need to support nodejs 8?

Reply:
I don't think so as its EOL for over two years now.

Reply:
👍  just merged https://github.com/dm-drogeriemarkt/lisa/pull/52